cfme/services/dashboard/ssui.py

```diff
@@ -81,33 +117,39 @@ def retiring_soon(self):
     view.flash.assert_no_error()
     view = self.create_view(MyServicesView)
     assert view.is_displayed
-    return retiring_services
+    view.paginator.set_items_per_page("100 items")
+    rows = view.paginator.items_amount
+    assert rows == retiring_services
 
 
 @Dashboard.current_services.external_implementation_for(ViaSSUI)
-def current_service(self):
+def current_services(self):
     """Returns the count of active services displayed on dashboard"""
 
     view = navigate_to(self, 'DashboardAll')
-    current_services = view.aggregate_card.get_count('Current Services')
+    current_service = view.aggregate_card.get_count('Current Services')
     view = navigate_to(self, 'CurrentServices')
     view.flash.assert_no_error()
     view = self.create_view(MyServicesView)
     assert view.is_displayed
-    return current_services
+    view.paginator.set_items_per_page("100 items")
+    rows = view.paginator.items_amount
+    assert rows == current_service
 
 
 @Dashboard.retired_services.external_implementation_for(ViaSSUI)
-def retired_service(self):
+def retired_services(self):
     """Returns the count of retired services displayed on dashboard"""
 
     view = navigate_to(self, 'DashboardAll')
-    retired_services = view.aggregate_card.get_count('Retired Services')
+    retired_service = view.aggregate_card.get_count('Retired Services')
     view = navigate_to(self, 'RetiredServices')
     view.flash.assert_no_error()
     view = self.create_view(MyServicesView)
     assert view.is_displayed
-    return retired_services
+    view.paginator.set_items_per_page("100 items")
+    rows = view.paginator.items_amount
+    assert rows == retired_service
```

Comment:
Same as with ``total_services``, consider if it would not be better to just return number of services.